services/catalog/tests/unit/conftest.py

```diff
@@ -268,8 +271,8 @@ async def rpc_client(
 
 
 @pytest.fixture
-def director_setup_disabled(monkeypatch: pytest.MonkeyPatch) -> None:
-    monkeypatch.setenv("CATALOG_DIRECTOR", "null")
+def director_setup_disabled(mocker: MockerFixture) -> None:
+    mocker.patch.object(simcore_service_catalog.core.events, "setup_director")
```

Comment:
Q: same here?

Reply:
Yes, i have changed that in all the external services. the catalog cannot work without any of these services connected.